### ISSUE TITLE
[Bug]: [CN-DN BVT] cn apply logtail with deletion failed

### ISSUE THREAD

**Issue by jiangxinmeng1** [matrixorigin/matrixone#5776]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

When update, logtail sent one insert and one delete. It failed and report duplicate, when receive insert first.

### Steps to Reproduce

```Markdown
Create database if not exists db1;
Use db1;
CREATE TABLE t1 (a int(3), b int(5) primary key);
INSERT INTO t1 VALUES (1,1),(1,2);
update t1 set a=2 where a=1 limit 1;
select * from t1;
```

**Comment by nnsgmsone** [comment]
I can't handle this, please make sure the log order. like (insert delete insert), not give me (insert insert delete)

**Comment by jiangxinmeng1** [comment]
Fixed by #5783
